tune_sklearn/utils.py

```diff
@@ -17,6 +17,14 @@ def check_warm_start(estimator):
             and is_not_tree_subclass)
 
 
+def check_warm_start_ensemble(estimator):
+    from sklearn.ensemble import BaseEnsemble
+    is_ensemble_subclass = issubclass(type(estimator), BaseEnsemble)
+
```

Comment:
should you update the above? `check_warm_start` can probably return True if is ensemble now

Reply:
I chose to separate it because warm start on non-ensemble estimators means we set `max_iter=1`, but warm start on ensemble estimators means we set `n_estimators=1`. I thought it'd be easier to handle these different cases with two separate checks.

Reply:
can you then rename the above to `can_warm_start_iter`?